pages/common/firefox.md

```diff
@@ -27,6 +27,14 @@
 
 `firefox --headless --screenshot {{path/to/output_file.png}} {{https://example.com/}}`
 
-- Use a specific profile directory to allow multiple separate instances of Firefox to run at once:
+- Use a specific profile directory to allow multiple separate instances of Firefox to run at once (not available for Windows):
```

Comment:
Well now this is interesting. I feel like we shouldn't have an example that is not available to all platforms in the common directory.
If that's the case we have two options:

Keep this `common` page, while creating a new page under `windows` but without this example.
Delete this page, recreate it under `linux` (and maybe `macOS` too?) while also creating a new page under `windows` without this example.
What's the way to go here, guys?

Reply:
Wait, this doesn't work for Windows? That _is_ surprising. What version of Firefox is that? If it's an old version it won't work, since it's a recent thing.

Also, I'd suggest we create a new page under Windows and keep the common one if we need to go down that route.

Reply:
I was digging through the [firefox cli docs](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#-new-instance), I'm not actually running the windows version. However I have just noticed that the functionality may be being removed as functionality, so it might be worth removing the flag in its entirety?

Reply:
It is? Do you have a reference for that? It would be highly unfortunate if that was the case, @scttrgd

Reply:
[This is the reference to it being removed as supported](https://bugzilla.mozilla.org/show_bug.cgi?id=855899#ct-37). But continuing to dig through conversations I'm not actually seeing it being removed, which is a little annoying. I've probably missed something from [this google group conversation](url). Apologies for muddying the waters here...

Reply:
Ah, I see. Thanks, @scttrgd! Reading into that, I see this:

> The new remoting does everything based on profile. When starting Firefox we
do normal profile selection, which includes considering any -P and
--profile command line arguments. Once we've selected a profile we attempt
to find an existing Firefox instance using that profile. If one is found we
send it our command line arguments and quit. If not continue start up. 

So to this end, we can instead remove the `--new-instance` flag, as in a future version of Firefox it will no longer be needed. However, I would prefer to make that change to tldr-pages once the change you linked to lands in stable.